packages/hash/frontend/src/pages/[account-slug]/[page-slug].page.tsx

```diff
@@ -17,6 +17,10 @@ import { useCollabPositions } from "../../blocks/page/collab/useCollabPositions"
 import { useCollabPositionTracking } from "../../blocks/page/collab/useCollabPositionTracking";
 import { PageBlock } from "../../blocks/page/PageBlock";
 import { PageTitle } from "../../blocks/page/PageTitle";
+import {
+  AccountPagesInfo,
+  useAccountPages,
+} from "../../components/hooks/useAccountPages";
 import { useArchivePage } from "../../components/hooks/useArchivePage";
```

Comment:
Haven't functionally tested, so am asking this Q somewhat blind... **are we losing the breadcrumbs entirely when a page is archived?**

I wonder if there's a better solution. e.g. showing the original location of the page in the tree (determined by its original parent's place in the hierarchy, irrespective of whether that parent may itself also now be archived), either grayed-out or explicitly labeled "ORIGINALLY LOCATED AT". @teenoh Want me to mock something up?

Reply:
Yeah we lose them. We currently use the `getAccountPages` query that populates the sidebar to generate the breadcrumbs and that query doesn't return archived pages. We don't use the `getPage` query to populate breadcrumbs because even though we have access to child/sub pages there, we don't have access to its parent pages. 
I wouldn't want to make any major backend changes so it doesn't conflict with work on the graph API, but I guess an easier (but not optimal) approach, will be to also return archived pages in `getAccountPages` query and filter them out in the sidebar. That way, we don't look breadcrumbs when a page is archived (since we won't hide archived pages in breadcrumbs)

Reply:
@teenoh Thanks. Do you know what happens when we restore a previously-archived page at the moment? (if we can?) Likely some follow-up work here as you flag.

Reply:
I missed this. If we restore a previously-archived page, the breadcrumbs come back since the page will now be included in what `getAccountPages` returns

Reply:
Teenoh says offline: "The breadcrumbs come back in that scenario"